bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BValueType.java

```diff
@@ -51,7 +53,17 @@
      */
     public abstract double floatValue();
 
+    /**
+     * Returns the value of the specified number as a {@code decimal},
+     * which may involve rounding.
+     *
+     * @return  the numeric value represented by this object after conversion
+     *          to type {@code decimal}.
```

Comment:
Shall we join this line with the previous line so that we could have the comment in a single line?

Reply:
Followed the same convention used for other methods of this abstract class.